packages/synapse-interface/constants/tokens/gasTokens.ts

```diff
@@ -0,0 +1,108 @@
+import { zeroAddress } from 'viem'
+import bscImg from '@assets/chains/bnb.svg'
+import metisImg from '@assets/chains/metis.svg'
+import moonbeamImg from '@assets/chains/moonbeam.svg'
+import moonriverImg from '@assets/chains/moonriver.svg'
+import harmonyImg from '@assets/chains/harmony.svg'
+import cantoImg from '@assets/chains/canto.svg'
+import dogechainImg from '@assets/chains/dogechain.svg'
+import cronosImg from '@assets/chains/cronos.svg'
+
+import * as CHAINS from '@/constants/chains/master'
+
+export type GasToken = {
+  addresses: { [x: number]: string }
+  chainId: number
+  decimals: number
+  symbol: string
+  name: string
+  icon: any
+}
+
+export const BNB: GasToken = {
+  addresses: {
+    [CHAINS.BNB.id]: zeroAddress,
+  },
+  chainId: 56,
```

Comment:
```
chainId: CHAINS.BNB.id
```

Reply:
Updated gasTokens constants to utilize object props defined in `CHAINS` constants in https://github.com/synapsecns/sanguine/pull/2401/commits/f3a0df19ffaac2512b4addc6b875f549128c5c19